llumnix/global_scheduler/migration_scheduler.py

```diff
@@ -24,9 +24,16 @@ class MigrationScheduler:
     def __init__(self,
                  pair_migration_policy: str,
                  migrate_out_load_threshold: float,
-                 instance_load_calculator: InstanceLoadCalculator) -> None:
+                 instance_load_calculator: InstanceLoadCalculator,
+                 migration_backend: str,) -> None:
         self.filter_config = MigrationFilterConfig(migrate_out_load_threshold=migrate_out_load_threshold)
         self.migration_filter = MigrationInstanceFilter(self.filter_config)
+        migration_backend_init_filter = CustomFilter()
```

Comment:
You can leave some comments here to explain what you are doing for here.

Reply:
get